types/recursive-readdir/v2/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for recursive-readdir 2.2
```

Comment:
you want to keep the latest version in `recursive-readdir/index.d.ts`. the older one is the one that moves. so you should have `v1/index.d.ts`.

Reply:
Fixed again (hopefully correctly this time).